src/TwigExtension.php

```diff
@@ -131,6 +146,10 @@ public function renderUrl(
      *
      * Usage: {{ absolute_url('path/to/something') }}
      * Generates: http://example.com/path/to/something
+     *
+     * @param string|null $path
+     *
+     * @return string
```

Comment:
Please revert the changes to this docblock.

We only document parameters and return values that:

- do not have typehints, and/or
- have additional narrative descriptions.

Neither `$path` or the return value fall in those categories.